driver/SCHEMA_VERSION

```diff
@@ -1 +1 @@
-2.23.0
+3.0.0
```

Comment:
As discussed here https://github.com/falcosecurity/libs/pull/2068#issuecomment-2507940752 we need a major bump in the SCHEMA VERSION because we are adding a new filler for `pread64` this should be an isolate case